Objective-C/LogSink/CBLLogSinks.mm

```diff
@@ -168,6 +185,51 @@ + (void) writeCBLLog: (C4LogDomain)c4domain level: (C4LogLevel)c4level message:
     [CBLLogSinks.custom writeLogWithLevel: level domain: domain message :message];
 }
 
+void writeCBLLogMessage(C4LogDomain domain, C4LogLevel level, NSString *msg, ...) {
```

Comment:
I think we should separate the responsibility of Log/Logger and LogSinks.
This should be at CBLLog.m/mm where the code uses for writing their log messages. Then CBLLog will use CBLLogSink(s) to forward to log to the enabled destination.